### ISSUE TITLE
The type 'T' must be a non-nullable value type in order to use it as parameter 'T' in the generic type or method 'np.array<T>(params T[])'	Numpy

### ISSUE THREAD

**Issue by sli-gpsw** [SciSharp/Numpy.NET#62]
The type 'string' must be a non-nullable value type in order to use it as parameter 'T' in the generic type or method 'np.array<T>(params T[])'	Numpy

![Capture2](https://user-images.githubusercontent.com/18432118/94500681-89e46880-0232-11eb-9659-e98793fbe717.PNG)

How to fix this？ thanks

**Comment by henon** [comment]
can you explain what you are trying to do? i.e. give me a python example of something that is not possible in numpy.net and I can add it for you.

**Comment by sli-gpsw** [comment]
Thanks Henon for reply. 
I programming in LabVIEW which provided by National Instrument company, called G language. LabVIEW can call .net via dll, then I can use the Numpy.Net in LabVIEW, 

But the Numpy.Net package has no compiled .dll files, so I need to compile the source code by myself, in the process, I met a few issues, the root cause is that I am not be familiar with VS2017.

could you please help me to compile Numpy.Net to dll? Thank you

**Comment by henon** [comment]
no need to compile yourself. just use the nuget packages. it is easy, you just have to go to your project dependencies open nuget package manager and search for the numpy.net package. If you don't know how to add nuget packages, please google

**Comment by sli-gpsw** [comment]
I figured out.
For LabVIEW user, for example, need below dll files wrapped by .NET.
Numpy.Bare.dll
Numpy.dll
Python.Deployment.dll
Python.Included.dll
Python.Runtime.dll

I just opened VS2019, I believe all other version should be fine.
Create a new console project, 
and run below lines under PM console

Install-Package Numpy -Version 3.7.1.23
Install-Package Numpy.Bare -Version 3.8.1.23

Then I build the project(selected release for any CPU)
go to output folder......\source\repos\ConsoleApp1\ConsoleApp1\bin\Release\netcoreapp3.1

then I can find above dll files now.
